js-extensions/EXTENSIONS.md

```diff
@@ -0,0 +1,62 @@
+# Jenkins JavaScript Extensions
+
+Jenkins JavasScript Extensions (JSe) in BlueOcean are handled in the UI by the `@jenkins-cd/js-extensions` module.
+
+JSe is based on the extensibility model already established by Jenkins, based on data and views, with the ability to inherit views based on parent data types.
+
+JSe `@jenkins-cd/js-extensions` module exports 2 things:
+- `store` - the `ExtensionStore` instance, which must be initialized
+- `Renderer` - a React component to conveniently render extensions
+
+### Store API
+
+The `ExtensionStore` API is very simple, all public methods are asynchronous:
+
+- `getExtensions(extensionPointName, [type,] onload)`
+    This method will async load data and type information as needed, and call the onload handler with a list of extension exports, e.g. the React classes or otherwise exported references.
+
+- `getTypeInfo(type, onload)`
```

Comment:
Does it really expose `getTypeInfo`? Does it need to? I'd think that should be a separate "Classes" api that extensions depends on.